GOVERNANCE.MD

```diff
@@ -1,6 +1,92 @@
 # Community Committee Governance
 
-The Community Committee governance is currently under pull request as the Community Committee Charter.
+Section 1. Guiding Principle.
+
+The Community Committee will operate transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to the entire ecosystem.
+
+Section 2. Evolution of Node.js Foundation Governance.
```

Comment:
It is my understanding that under current governance and how we are setting this up, the CC and the TSC would be siblings. Anything that one is doing that can affect other parts of the project, should under all circumstances first be discussed with one another in community and leadership. If, after due diligence, discussions, and vote, the orgs are not able to come into agreement, the Board would be brought in for advising.

But! These are all from conversations and the understanding I've been trying to grasp myself as writing this. I think @williamkapke might be able to chime in on the existing structure and maybe how we can imagine this going healthily.